pandas/io/formats/style.py

```diff
@@ -1072,6 +1072,147 @@ def hide_columns(self, subset) -> Styler:
     # A collection of "builtin" styles
     # -----------------------------------------------------------------------
 
+    def text_gradient(
+        self,
+        cmap="PuBu",
+        low: float = 0,
+        high: float = 0,
+        axis: Axis | None = 0,
+        subset=None,
+        vmin: float | None = None,
+        vmax: float | None = None,
+        gmap: Sequence | None = None,
+    ) -> Styler:
+        """
+        Color the text in a gradient style.
+
+        The text color is determined according
+        to the data in each column, row or frame, or by a given
+        gradient map. Requires matplotlib.
+
+        .. versionadded:: 1.3.0
+
+        Parameters
+        ----------
+        cmap : str or colormap
+            Matplotlib colormap.
```

Comment:
these match the kwargs for background_gradient? if not why not

Reply:
can you use a shared doc-string?

Reply:
yes they all match its the same signature. Can you use a shared doc string even if the short summary part is different? I've never used a shared doc string.. any tips where one is used that I can lookup and copy?

Reply:
nevermind.. i figured it out and made the changes.